platform/platform-app/pom.xml

```diff
@@ -389,33 +390,36 @@
         <cxf.opensaml.osgi.version.range>[3.1,4)</cxf.opensaml.osgi.version.range>
         <cxf.rhino.version>1.7R2</cxf.rhino.version>
         <cxf.reflections.bundle.version>0.9.10_3</cxf.reflections.bundle.version>
+        <!--<cxf.servlet-api.group>org.apache.geronimo.specs</cxf.servlet-api.group>-->
         <cxf.servlet-api.group>org.apache.geronimo.specs</cxf.servlet-api.group>
         <cxf.servlet-api.artifact>geronimo-servlet_3.0_spec</cxf.servlet-api.artifact>
         <cxf.servlet-api.version>1.0</cxf.servlet-api.version>
         <cxf.servlet-api-2.5.artifact>geronimo-servlet_2.5_spec</cxf.servlet-api-2.5.artifact>
         <cxf.servlet-api-2.5.version>1.1.2</cxf.servlet-api-2.5.version>
-        <cxf.slf4j.version>1.7.21</cxf.slf4j.version>
+        <cxf.slf4j.version>1.7.22</cxf.slf4j.version>
         <cxf.specs.jaxws.api.version>1.2</cxf.specs.jaxws.api.version>
-        <cxf.spring.version>${spring.version}</cxf.spring.version>
-        <cxf.spring.security.version>3.2.9.RELEASE</cxf.spring.security.version>
+        <cxf.spring.version>4.1.9.RELEASE</cxf.spring.version>
```

Comment:
This version isn't used in `platform/platform-app/src/main/resources/features.xml`, so it shouldn't need to be updated here.